symposion/speakers/forms.py

```diff
@@ -17,12 +17,42 @@ class Meta:
             "photo",
             "twitter_username",
             "mobile_number",
+            "interested_mentee",
+            "interested_mentor",
+            "financial_support",
         ]
         help_texts = {
             'mobile_number': _(
                 u"For on-site use only,"
                 u" in case we need to get in touch with you."
             ),
+            'interested_mentee': _(
```

Comment:
@Mariatta any suggestions for good information to put in these help texts?

The tl;dr I'm looking for is "check these boxes and we'll get back to you as soon as we have more information on this program, but I want it to sound more inviting than that" 🙁

Reply:
Thanks @ewdurbin! So exciting that this is happening!

I like the different choices you have here. While running the PyCascades speaker mentorship program, we found some submitters were only interested in mentorship if their proposal is accepted (e.g. wanting feedback about slides). Some others would like more feedback at proposal preparation stage.

I think it's not necessary to mention that this is something "new".

What do you think about "We want to support our speakers. Let us know the different ways we can help you, and will get back once we have more information"

Maybe you can also mention that this is not limited to only new/first time speakers.

Reply:
@Mariatta thank you so much for your input, it's been invaluable! do you mind if I request a review of my draft when it is complete?

Reply:
I think we can follow up with another effort specifically for mentorship around preparation/practice _after_ talks are accepted!

Reply:
For sure!

Reply:
Alright @Mariatta updated copy for mentors/mentees! Any/all feedback welcome :)

Reply:
Looks good to me! Thanks :)